integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -142,7 +142,7 @@ public void lineageCompleteAndAsyncPersistTest() throws Exception {
           fileInfo.getPersistenceState());
 
       // sleep and wait for worker to persist the file
-      CommonUtils.sleepMs(5);
+      CommonUtils.sleepMs(200);
```

Comment:
Currently we create a new thread per file, so that the persistence can do it in parallel. Also, if a thread fails, it doesn't affect the ongoing persistence of others.